angular-app/src/app/project-detail/project-detail.component.html

```diff
@@ -1,23 +1,30 @@
 <div *ngIf="project$ | async as project">
-  <mat-toolbar>
-    <span>Project: {{project.title}}</span>
-    <span class="spacer"></span>
-    <button mat-icon-button matTooltip="Project Settings" (click)="openDialog()">
-      <mat-icon>settings</mat-icon>
-    </button>
-    <button mat-icon-button (click)="toggleChat()" aria-label="toggle chat" matTooltip="Toggle Chat">
-      <mat-icon>chat</mat-icon>
-    </button>
-  </mat-toolbar>
-  <mat-selection-list [multiple]="false">
-    <mat-list-option *ngFor="let file of files">
-      <div class="list-item-content">
-        <span>{{file}}</span>
-        <span class="spacer"></span>
-        <button mat-icon-button matTooltip="Open in Workspace">
-          <mat-icon>edit</mat-icon>
-        </button>
+  <mat-tab-group animationDuration="0ms">
```

Comment:
Does 0ms mean there is no animation? What is the default animationDuration?

Reply:
Yes, this is the official way to "remove" animation. I'm not sure what the default is (except nonzero and less than 1000ms), but there is a way to make 0ms the default, which I'm happy to do.

Reply:
SG. Thanks for clarification.